extensions/amp-story/1.0/test/test-amp-story-store-service.js

```diff
@@ -172,6 +172,28 @@ describes.fakeWin('amp-story-store-service actions', {}, env => {
     expect(pausedListenerSpy).to.have.been.calledWith(true);
   });
 
+  it('should pause the story when displaying the bookend', () => {
+    const pausedListenerSpy = sandbox.spy();
+    storeService.subscribe(StateProperty.PAUSED_STATE, pausedListenerSpy);
+    storeService.dispatch(Action.TOGGLE_BOOKEND, true);
+    expect(pausedListenerSpy).to.have.been.calledOnce;
+    expect(pausedListenerSpy).to.have.been.calledWith(true);
+  });
+
+  it('should unpause the story when closing the bookend', () => {
+    const pausedListenerSpy = sandbox.spy();
+    storeService.subscribe(StateProperty.PAUSED_STATE, pausedListenerSpy);
+
+    // First open the bookend.
+    storeService.dispatch(Action.TOGGLE_BOOKEND, true);
+
+    // Close the bookend.
+    storeService.dispatch(Action.TOGGLE_BOOKEND, false);
+
+    expect(pausedListenerSpy).to.have.been.calledTwice;
+    expect(pausedListenerSpy).to.have.been.calledWith(false);
+  });
```

Comment:
Nit: you could register the `pausedListenerSpy` after you open the bookend, before you close it :)